src/components/crud-form-pages/Clusters.js

```diff
@@ -18,6 +18,7 @@ export const Clusters = (props) => {
       forceOpen={props.forceOpen}
       docs={props.docs}
       innerFormProps={{
+        noSecretsManager: props.noSecretsManager,
```

Comment:
`noSecretsManager` prop os not needed since `secretManagerList` is already being passed. instead of using `props.noSecretsManager` use `isEmpty(secretsManagerList)`